Refresh.GameServer/Database/GameDatabaseContext.Relations.cs

```diff
@@ -192,6 +192,81 @@ public void ClearQueue(GameUser user)
 
     #region Rating and Reviewing
 
+    public void RateReview(GameReview review, RatingType rt, GameUser user)
+    {
+        // if rating type is neutral, remove previous review rating by user 
+        if (rt == RatingType.Neutral && this.ReviewRatingExistsByUser(user, review))
+        {
+            RateReviewRelation rel =
+                this.RateReviewRelations.First(relation => relation.Review == review && relation.User == user);
```

Comment:
...but for LINQ we typically do first character, so `r => ...` in this case. `x => ...` is also a common pattern I see.